### PR TITLE
E2E tests

### PR DESCRIPTION
**Pull request by lkostrowski**
## Scope of the PR

1. Add package with configured e2e with playwright (packages/e2e)
2. Add some smoke tests to apps

Not ready to be run on CICD but can be run locally before releases to run some checks

## Checklist

- [ ] `.github/dependabot.yaml` is up-to date.
- [ ] I added changesets and [read good practices](/.changeset/README.md).

### CODE REVIEW

File: packages/e2e/setup/routing.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { configuration } from "./configuration";
+
+export const appUrls = (appUrl: string) => ({
+  manifest: new URL("/api/manifest", appUrl).href,
+  register: new URL("/api/register", appUrl).href,
+});
+
+export const saleorUrls = (dashboardUrl: string) => ({
```
Comment (peelar) [inline packages/e2e/setup/routing.ts:8]:
```suggestion
const saleorUrls = (dashboardUrl: string) => ({
```

nitpick, but I don't think it has to be exported

File: packages/e2e/tests/apps-installation.spec.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import { test, expect, Page } from "@playwright/test";
+import { logInIntoDashboard } from "./operations/log-in-to-dashboard";
+import { installTheApp } from "./operations/install-app";
+import { appUrls, routing } from "../setup/routing";
+import { AppManifest } from "@saleor/app-sdk/types";
+import { assertAppAvailable } from "./assertions/assert-app-available";
+
+// target every app including staging. This should be config from the outside
```
Comment (peelar) [inline packages/e2e/tests/apps-installation.spec.ts:8]:
```suggestion
// target every app including staging. This should be config from the outside
```

What do you mean by "the outside"? Can you specify? Is this a todo comment for the future or a note?

File: packages/e2e/tests/apps/product-feed/product-feed-configuration.spec.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    await expect(page.getByText("Updated S3 configuration")).toBeVisible({ timeout: 10000 });
+
+    await fillChannelConfig(iframeLocator);
+
+    await expect(page.getByText("Success")).toBeVisible({
+      timeout: 10000,
+    }); // todo add more meaningul message, only "success" is set
+  });
+
+  test.skip("App can be configured with categories mapping", async () => {
```
Comment (peelar) [inline packages/e2e/tests/apps/product-feed/product-feed-configuration.spec.ts:45]: please add a comment for why this one is skipped

File: packages/e2e/playwright.config.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+require("dotenv").config();
```
Comment (peelar) [inline packages/e2e/playwright.config.ts:7]: not sure if it's needed, but we can validate if the required env vars were provided

**Review by peelar — APPROVED** [review]

**Review by krzysztofwolski — APPROVED** [review]
Don't see anything preventing us from merging. Would like to start adding data attrs to EAM app and start writing test.

File: packages/e2e/playwright.config.ts (hunk 4 above)
Comment (lkostrowski) [reply packages/e2e/playwright.config.ts:7]: Its needed and its validated. Added comment where to look it up

**Pull request merged**